introduction.md

```diff
@@ -0,0 +1 @@
+My name's Yuri Leite and I'm a mighty pirat...err, ninja!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.